Added initial documentation set

tickbox-smc opened the pull request:
The additional documentation set has been added as the initial template
structure for documents.

Accepting that this is just a starter, can you have a look at the documents and let me know what you think

@jaynebickerton
@stevecarruthers

Commit:
Added initial documentation set

The additional documentation set has been added as the initial template
structure for documents.

stevecarruthers commented on README.md:
```diff
@@ -1,4 +1,35 @@
+## Why xfero?
+**THE MISSING LINK IN BUSINESS FILE TRANSFER SERVICES**
+
+Xfero is the complete orchestration framework for business file transfer services with a mission to simplify complex file transfer integration while giving business users the ability to manage configuration rather than relying on techies.
```
xfero is a complete orchestration framework that simplifies complex file transfer integration for business file transfer services whilst giving the business community the ability to manage these services without having to engage with their technical department.

Standardise the name as xfero regardless of grammar ?

tickbox-smc replied:
Hi Steve, thanks for the comments. :+1: Do you intend the paragraph you shared to replace the one I had? And also, are you recommending that we use 'xfero' as opposed to 'Xfero' or 'XFERO'. If so I'd like to hear what @jaynebickerton thinks as our English and punctuation expert

Jayne, any thoughts... :smiley:

stevecarruthers replied:
XferO

jaynebickerton replied:
So, the pedant in me has never liked starting a sentence with a lower case letter but it is something I've got used to, so my view isn't based on grammatical correctness but, when I've been reading the wiki pages I have found 'Xfero' more eye catching when it occurs in the middle of a sentence, e.g. "...removes the complexities of configuration via Xfero Navigator"

jaynebickerton replied:
XferO is quite aesthetically pleasing 😄

stevecarruthers replied:
👍

stevecarruthers commented on README.md:
```diff
@@ -1,4 +1,35 @@
+## Whats different?
+The primary distinguishing features of xfero are:
+
+- Xfero puts the business user in charge. Providing a simple GUI for configuration of transfer routing.
```
business user -->  business community

tickbox-smc committed:
Replaced the Why XferO sentence

Following conversation in #2 it was agreed that the original wording of this sentence could be improved. This led to longer discussion about the name and how it should be written. Decided to use XferO :+1:

tickbox-smc committed:
Replaces the 'business user' statement 

'business user' -> 'business community' which is a more appropriate phrase. #2

stevecarruthers commented on README.md:
```diff
@@ -1,4 +1,35 @@
+## Whats different?
+The primary distinguishing features of xfero are:
+
+- Xfero puts the business community in charge. Providing a simple GUI for configuration of transfer routing.
```
Replace with ... "XferO puts the business community in charge by providing a simple GUI for configuring and managing transfer routes."

Need to replace all with XferO if that is the chosen name.

Does this impact the code ?
Change the name of the "whatever xfero is in GitHub".

tickbox-smc replied:
Agreed will fix this, thanks

tickbox-smc committed:
Modified sentence structure

Following discussion with @stevecarruthers we agreed to change sentence structure of the bullet for better readability
